ingestion-beam/pom.xml

```diff
@@ -17,6 +17,21 @@
         <exec.mainClass>com.mozilla.telemetry.Sink</exec.mainClass>
     </properties>
 
+    <profiles>
+        <profile>
+            <!-- Enable this profile to add ingestion-core as a dependency for IDEs like Intellij
+                 that don't detect files added by build-helper-maven-plugin -->
+            <id>IDE</id>
+            <dependencies>
+                <dependency>
+                    <groupId>${project.groupId}</groupId>
+                    <artifactId>ingestion-core</artifactId>
+                    <version>${project.version}</version>
+                </dependency>
+            </dependencies>
+        </profile>
+    </profiles>
+
```

Comment:
Is this block intended as part of this PR?

Reply:
nope, it got rebased out, i'm fixing tests and found that i was going to create hard-to-decipher conflicts with #1117